webpack.config.js

```diff
@@ -139,18 +148,24 @@ module.exports = function makeWebpackConfig () {
    * Reference: https://github.com/postcss/autoprefixer-core
    * Add vendor prefixes to your css
    */
-  config.postcss = [
-    autoprefixer({
-      browsers: ['last 2 version']
-    })
-  ];
+   // NOTE: This is now handled in the `postcss.config.js`
+   //       webpack2 has some issues, making the config file necessary
```

Comment:
Like I mentioned above, I will track down where the information I gleaned as to requiring the `postcss.config.js`   I understand what you are asking, but I am just stating that I dont recall where I say this (when I had to add it to my project).

Further, adding the `postcss.config.js` is the usage of `.babelrc` (it too could be stored in the `package.json`

I will track down where I found the information about needing to use `postcss.config.js` with webpack 2